vmdk_plugin/plugin.go

```diff
@@ -209,6 +221,10 @@ func (d *vmdkDriver) unmountVolume(name string) error {
 // Name and driver specific options passed through to the ESX host
 func (d *vmdkDriver) Create(r volume.Request) volume.Response {
 
+	// Lock this volume for create
+	d.lockVolume(r.Name)
+	defer d.unlockVolume(r.Name)
```

Comment:
This will essentially create a refmap entry and lock the entry. Once create completes the lock is released and the refmap entry will be removed once create completes. Do we then need the lock for the create? Same for remove, the last unmount will destroy the refmap entry and remove creates and deletes it.